### PR TITLE
Expand file globs

### LINKED ISSUE #210: Zuul mishandles file globs
The use of `path.resolve` [here](https://github.com/defunctzombie/zuul/blob/master/lib/control-app.js#L34) will not properly expand valid file globs. Example: `test/**/*.js`

This causes zuul to log errors whenever you try to run tests using wildcard subdirectory globs

### PR DESCRIPTION
**Pull request by chinedufn**
Fixes #210 

Let me know if anything needs to be changed/fixed!

### CODE REVIEW

**Comment by chinedufn** [comment]
Wrote a small tested package to handle this - https://github.com/chinedufn/globs-to-files/blob/master/src/index.js

If this gets the OK I'll make a new commit using it then squash

File: lib/control-app.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 var path = require('path');
 var fs = require('fs');
+var glob = require('glob')
```
Comment (defunctzombie) [inline lib/control-app.js]: no semicolon

File: package.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
     "express-state": "1.0.3",
     "find-nearest-file": "1.0.0",
     "firefox-profile": "0.2.7",
+    "glob": "^5.0.10",
```
Comment (defunctzombie) [inline package.json]: notice how the other dependencies have their versions specified

File: lib/control-app.js [hunk 3]
```diff
@@ -30,9 +31,13 @@ module.exports = function(config) {
         fullPaths: true
     });
 
-    files = files.map(function(file) {
-        return path.resolve(file);
+    var expandedFiles = [];
+    files.forEach(function (arg) {
```
Comment (defunctzombie) [inline lib/control-app.js]: could use `.map` ?

Comment (chinedufn) [inline lib/control-app.js]: Could also use map.

I'd rather change this into one line of tested code using [globs-to-files](https://github.com/chinedufn/globs-to-files/blob/master/index.js#L33-L36) and then `var files = deglob.sync(files)`

But let me know what you prefer and I'll do it :)

Comment (defunctzombie) [inline lib/control-app.js]: have no strong preference there, using a tested lib seems good :)

Once you have the code how you like, squash the commits and ping me for the PR again

**Comment by chinedufn** [comment]
@defunctzombie squashed. Let me know if anything needs fixing!

**Comment by defunctzombie** [comment]
@chinedufn tests are not passing

**Comment by chinedufn** [comment]
@defunctzombie just committed and squashed a fix that *should* do the job (wasn't expanding globs synchronously)

From glancing at some of the recently merged PR's it looks like I can safely ignore my `Sauce Labs Authentication Error` ?

Let me know if there are any more issues!

**Comment by vvo** [comment]
Yes the saucelabs credentials are encrypted so tests cannot run in travis on a PR

**Pull request merged**